edx-platform/msus-produhktiv/lms/templates/index.html

```diff
@@ -18,7 +19,7 @@
                 ${homepage_overlay_html | n, decode.utf8}
               % else:
                   ## Translators: 'Open edX' is a registered trademark, please keep this untranslated. See http://open.edx.org for more information.
-                  <h1>${Text(_(u"Welcome to the Microsoft U.S. Learning Portal"))}</h1>
+                  <h1>${Text(_(u"Welcome to the {portal_name}")).format(portal_name=configuration_helpers.get_value('PORTAL_NAME', settings.PORTAL_NAME))}</h1>
```

Comment:
why don't you use PLATFORM_NAME instead of creating a custom setting?

Reply:
right! forgot that, sorry

Reply:
because PLATFORM_NAME has a value like msus.produhktivx.com instead of the portal title, and I'm not sure if that's being used elsewhere @jfavellar90

Reply:
Yes, I think that settings.PORTAL_NAME is undefined, it is not throwing errors?

Reply:
not locally @morenol

Reply:
Sorry I didn't get the question at first, it gives an error if the portal_name setting is undefined, going to fix that now so it takes the platform name by default. Thanks for the suggestion @morenol